jruby-gradle-base-plugin/build.gradle

```diff
@@ -39,7 +47,7 @@ dependencies {
         exclude module : 'groovy-all'
     }
 
-    provided "org.jruby:jruby-complete:${jrubyVersion}"
+    compile "org.jruby:jruby-complete:${jrubyVersion}"
```

Comment:
Could you provide some commentary either here or in the code as to why this dependency change is needed? I thought the rubygems war was all in Java?